doc/01-getting-started.md

```diff
@@ -1,81 +1,93 @@
 # Getting Started
```

Comment:
This is mostly just VS Code doing its formatting thing.  But I did update the Hyperproof CLI links to point to the local docs.  I think that eventually the CLI docs move elsewhere, but this will work for now.